llvm/test/CodeGen/SPIRV/gpu_intrinsics.ll

```diff
@@ -0,0 +1,427 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-attributes
+; RUN: opt -S -mtriple=amdgcn-- -passes=lower-gpu-intrinsic < %s | FileCheck %s --check-prefix=AMDGCN
```

Comment:
Should not be testing amdgpu handling in test/CodeGen/SPIRV. Duplicate the test.

Also this is just an IR->IR test, should go in test/Transforms/LowerGPUIntrinsics

Reply:
I don't love copy&paste the test since the input IR is identical between the two and they'll definitely drift out of sync but sure, I'll copy&paste the thing as requested. Dumping them in a new directory to slightly improve the chances that people editing one remember to edit the other.